plugins/modules/system/java_cert.py

```diff
@@ -81,9 +82,12 @@
   state:
     description:
       - Defines action which can be either certificate import or removal.
+      - When state is present, the certificate will always idempotently be inserted
+      - into the keystore, even if there already exists a cert alias that is different.
```

Comment:
This isn't a new paragraph, but continues the previous sentence:
```suggestion
        into the keystore, even if there already exists a cert alias that is different.
```